### ISSUE TITLE
Incorrect human_attribute_name in case of single table inheritance (STI)

### ISSUE THREAD

**Issue by kalsan** [grosser/gettext_i18n_rails#200]
Hi!

The following is a bug report. When calling `.human_attribute_name(...)` on an STI child class in Rails, `gettext_i18n_rails` looks up the wrong constant. Here's an example:

```ruby
class Parent < ApplicationRecord; end
class Child < Parent; end
```

Assuming that `Parent` has a field `:foo` which it shares with `Child` via STI. Translations are (`app.po`):

```
msgid "Parent|Foo"
msgstr "Parent's Foo"

msgid "Child|Foo"
msgstr "Child's Foo"
```

Expected:

```ruby
Parent.human_attribute_name(:foo) # Parent's foo
Child.human_attribute_name(:foo) # Child's foo
```

However, `gettext_i18n_rails` returns:

```ruby
Parent.human_attribute_name(:foo) # Parent's foo
Child.human_attribute_name(:foo) # Parent's foo --> WRONG
```

Best,
Kalsan

**Comment by grosser** [comment]
yeah kinda bad ... PR welcome, ideally this would be a fallback from `Child|foo` to `Parent|foo` to not break existing behavior, or an opt-in

**Comment by kalsan** [comment]
@grosser implemented it with a fallback and created pull request #208 at your convenience. Looking forward to hearing your thoughts and/or merge :-)

Have a good day!
Kalsan
